### ISSUE TITLE
MaximalSubgroups errors for OmegaPlus(8,4)

### ISSUE THREAD

**Issue by ssiccha** [gap-system/gap#4614]
I get the following on the current master:
```
gap> grp := OmegaPlus(8,4);
<matrix group of size 67010895544320000 with 2 generators>
gap> maxes := MaximalSubgroups(grp);; time;
Error, Variable: 'H' must have an assigned value in
  m := TryMaximalSubgroupClassReps( H : cheap := false,
    intersize := false,
    inmax := false,
    nolattice := false ); at /home/sergio/.local/opt/gap/gap-master/lib/grp.gi:1465 called from
MaximalSubgroupClassReps( G ) at /home/sergio/.local/opt/gap/gap-master/lib/grplatt.gi:1805 called from
ConjugacyClassesMaximalSubgroups( G ) at /home/sergio/.local/opt/gap/gap-master/lib/grplatt.gi:1815 called from
```

**Comment by jackschmidt** [comment]
I think this was fixed in c8cd5b675fb293f08ce7812c74b3e81fc073fedf

**Comment by hulpke** [comment]
I close this since it has been fixed in #4610
